net/obfs4proxy/Makefile

```diff
@@ -8,14 +8,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=obfs4proxy
-PKG_VERSION:=0.0.7
+PKG_VERSION:=0.0.8
 PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_URL:=https://git.torproject.org/pluggable-transports/obfs4.git
-PKG_SOURCE_VERSION:=8256fac93c2cf79742725e3aaced5bbe3380fd32
-PKG_SOURCE_DATE:=20161115
-PKG_MIRROR_HASH:=9c504337f7a2b46310083224cf0998fc7645f8af5d50657cf74358d39916eddc
+PKG_SOURCE_URL:=https://gitlab.com/yawning/obfs4.git
```

Comment:
https://github.com/Yawning/obfs4
Use codeload

Reply:
ok, no problem. Is there any particular reason why is GitHub preferred over Gitlab ?

Reply:
Tarballs are preferred over git/svn etc

Reply:
changed to codeload